src/core/MOM_forcing_type.F90

```diff
@@ -631,12 +634,9 @@ subroutine extractFluxes1d(G, GV, US, fluxes, optics, nsw, j, dt, &
     if (fluxes%evap(i,j) < 0.0) netMassOut(i) = netMassOut(i) + fluxes%evap(i,j)
   !   if (associated(fluxes%heat_content_cond)) fluxes%heat_content_cond(i,j) = 0.0 !??? --AJA
 
-    ! lprec < 0 means sea ice formation taking water from the ocean.
-    ! smg: we should split the ice melt/formation from the lprec
-    if (fluxes%lprec(i,j) < 0.0) netMassOut(i) = netMassOut(i) + fluxes%lprec(i,j)
```

Comment:
The changes in the next ~8 lines will change answers for people who are using the NUOPC coupler and are already using `fluxes%seaice_melt`.  A runtime flag might be needed here to recreate previous answers.